### PR TITLE
Add booked visit totals to the view hospital page

### PR DESCRIPTION
**Pull request by jivdhaliwal**
# What
Adds the total booked visits to the view hospital page

# Why
Helps breakdown the usage across hospitals

# Screenshots
localhost_3000_trust-admin_hospitals_2(iPad)

# Notes

### CODE REVIEW

File: src/usecases/retrieveHospitalVisitTotals.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      JOIN wards ON wards.id = totals.ward_id
+      JOIN hospitals ON wards.hospital_id = hospitals.id
+    WHERE hospitals.trust_id = $1
+    GROUP BY hospitals.id`,
+    [trustId]
+  );
+  let response = {};
+
+  queryResult.map(({ hospital_id, total_visits }) => {
+    response[hospital_id] = total_visits;
```
Comment (chubberlisk) [inline src/usecases/retrieveHospitalVisitTotals.js:18]: In `src/usecases/retrieveWardVisitTotals.js`, we parse the number to as an int. Could we do the same here to be consistent?

File: pageTests/trust-admin/hospitals/[id].test.js [hunk 2]
```diff
@@ -49,6 +49,7 @@ describe("trust-admin/hospitals/[id]", () => {
       const container = {
         getRetrieveWardsByHospitalId: () => wardsSpy,
         getRetrieveHospitalById: () => hospitalSpy,
+        getRetrieveHospitalVisitTotals: () => jest.fn().mockReturnValue({}),
```
Comment (chubberlisk) [inline pageTests/trust-admin/hospitals/[id].test.js:52]: Could we add an expectation somewhere to check we're calling this usecase with the trust ID and that it gets set as props?

File: src/usecases/retrieveHospitalVisitTotals.contractTest.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
+import AppContainer from "../containers/AppContainer";
+
+describe("retrieveHospitalVisitTotals contract tests", () => {
+  const container = AppContainer.getInstance();
+
+  it("returns the total number of visits for each hospital", async () => {
+    const { trustId } = await container.getCreateTrust(container)({
```
Comment (chubberlisk) [inline src/usecases/retrieveHospitalVisitTotals.contractTest.js:7]: Do we need the `container` to be passed into `getCreateTrust`?

File: pageTests/trust-admin/hospitals/[id].test.js (hunk 2 above)
Comment (jivdhaliwal) [reply pageTests/trust-admin/hospitals/[id].test.js:52]: Ye good point 👍

File: src/usecases/retrieveHospitalVisitTotals.contractTest.js (hunk 3 above)
Comment (jivdhaliwal) [reply src/usecases/retrieveHospitalVisitTotals.contractTest.js:7]: good catch thanks

**Pull request merged**